catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/source/SierraBibData.scala

```diff
@@ -3,8 +3,8 @@ package uk.ac.wellcome.transformer.source
 case class SierraBibData(
   id: String,
   title: String,
-  deleted: Boolean,
-  suppressed: Boolean,
+  deleted: Boolean = false,
+  suppressed: Boolean = false,
```

Comment:
This is because circe will fail decoding json that doesn't contain those fields unless there is a default value specified

Reply:
When would we not have a value?